_extras/checkout.md

```diff
@@ -219,13 +219,14 @@ think your lesson has good start points for participatory instruction, you may e
 work for a checkout demo. Be sure to give yourself time to change course if your planned lesson will not work.
 
 ### Sign-up and Set-up
-To sign up, select a session that works for you on the [the teaching demo schedule Etherpad]({{page.demopad}}), and add
-your name and a link to your lesson of choice to the Etherpad. Be sure to **double check the time in your local time zone** by clicking on the converter
+To sign up, select a session that works for you on [the Training Demonstration Sessions Etherpad]({{page.demopad}}), and add
+your name and a link to your lesson of choice to that Etherpad. Be sure to **double check the time in your local time zone** by clicking on the converter
```

Comment:
```suggestion
To sign up, select a session that works for you on [the Teaching Demo Etherpad]({{page.demopad}}), and add
your name and a link to your lesson of choice to an open slot. Be sure to **double check the time in your local time zone** by clicking on the converter
```
We should use the name here that matches, as best as possible how trainers will refer to that pad, and what it it titled.

Reply:
Re line 222, the current title of the document is "The Carpentries Instructor Training Teaching Demonstration Sessions". "Teaching Demo Etherpad" is confusing until that becomes the title of the document.

Re line 223, makes sense.